waiter/config-full.edn

```diff
@@ -484,6 +481,15 @@
                            ;; The number of hosts on which health checks must fail in order to declare a deployment error:
                            :min-hosts 2}
 
+ ;; Sets tolerance level for Waiter regarding health checks
```

Comment:
Not sure how helpful this comment is given that there are solid comments on each of the sub-elements